tests/interop/main.rs

```diff
@@ -123,7 +123,7 @@ fn start_js_server() -> Child {
 
 fn npm_build() {
     println!("npm run build");
-    let mut cmd = std::process::Command::new("/usr/bin/npm");
+    let mut cmd = std::process::Command::new("npm");
```

Comment:
Hardcoding npm path doesn't work in many cases. For example when using https://github.com/Schniz/fnm

Reply:
Yeah good spot, I fixed this in #29